.pre-commit-config.yaml

```diff
@@ -12,8 +12,8 @@ repos:
       - id: check-symlinks
       - id: debug-statements
       - id: detect-private-key
-      - id: end-of-file-fixer
-      - id: trailing-whitespace
+      # - id: end-of-file-fixer
```

Comment:
Why are these removed?  Don't we want these checks?  And why comment them out, instead of deleting them?

Reply:
Doh, I had clicked auto-merge so this got merged as soon as you approved it. I can follow this up with a PR to remove them if that's what you'd like.

I commented them out because these are all failing and likely have been for some time. The PR that enabled them had a bug in the workflow where it ran the checks but never supplied any files to check (so it was checking nothing). See https://github.com/spacetelescope/stcal/pull/293#discussion_r1783271648 for a comment on where this bug is fixed. Here's a PR that uncomments all the checks to show the extent of the failures:
https://github.com/spacetelescope/stcal/pull/294
The reported diff for the checks is over 5000 lines long:
https://github.com/spacetelescope/stcal/actions/runs/11142692931/job/30966194873?pr=294
(and doesn't fix all the failures and many require manual updates).

I chose to not remove them because I assumed that the maintainers want these checks to run. If that's not the case I'd say we remove them entirely. If the re-enabling the checks are preferred I was thinking it would be easiest to enable them perhaps one-by-one since each will require file changes.

Reply:
I'm not sure what's the best approach for this.  It could be good to have them commented out, so we have explicit documentation for what is not used, but removing them to clean up the script could be good, too.  I just wanted to make sure you wanted the comments left in the scripts.